File: Typescript/test/!template.test.ts
import { expect, test } from '@jest/globals';
import * as solution from '../src/minCosts';

test('Example 1', () => {
	let input = [5,3,4,1,3,2];
	let expected = [5,3,3,1,1,1];
	expect(solution.minCosts(input)).toEqual(expected);
});

File: Dart/test/!template_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:leetcode/getSneakyNumbers.dart';

void main() {
  test("test example", () {
    var input = [0, 1, 1, 0];
    var expected = true;

    final sln = Solution();
    var output = sln.getSneakyNumbers(input);
    expect(output, expected);
  });
}
